component/badge/badge.css

```diff
@@ -1,58 +1,14 @@
- @import url('/style.css');
- @import url('/component/slidebaar.css');
-
- 
  .header-content{ 
     height: 5rem;
     width: 100%;
     background-color: var(--secondary-color);
-}
- h5{
-    border: 1px  solid;
-    border-radius: 50%;
-    font-size: 1rem;
-    padding: 1px 0 2px 2px;
-    padding-right: 0.8rem;
-    width: 0.5px;
-    margin-top: -3rem;
-    position: relative;
-    background-color:  var(--primary-color);
- }
- 
-.badge-icon{
-    height: 20%;
-    width: 90%;
-   justify-content: center;
-    border: 1px solid ;
- }
-.badge{
-    margin-top: 4rem;
-}
-.symbol3{
-    margin-left: 1.7rem;
-}
-.symbol4{
-    margin-left: 2.1rem;
 }
 .avator-badge img{
```

Comment:
better to use dedicated classes IMO as project gets bigger, it may complicate things.

Reply:
I have resolve code